src/TelemetryChannels/ServerTelemetryChannel/NuGet/Package.nuspec

```diff
@@ -6,11 +6,11 @@
     <title>Application Insights Telemetry Channel for Windows Server Applications</title>
     <authors>$author$</authors>
     <owners>$author$</owners>
-    <licenseUrl>http://go.microsoft.com/fwlink/?LinkID=391182</licenseUrl>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkID=510709</licenseUrl>
     <projectUrl>http://go.microsoft.com/fwlink/?LinkId=392727</projectUrl>
     <iconUrl>$image$</iconUrl>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <description>This nuget provides a telemetry channel to Application Insights Windows Server SDK that will preserve telemetry in offline scenarios. This is a dependent package, for the best experience please install the platform specific package.</description>
+    <description>This nuget provides a telemetry channel to Application Insights Windows Server SDK that will preserve telemetry in offline scenarios. This is a dependent package, for the best experience please install the platform specific package. Privacy statement: https://go.microsoft.com/fwlink/?LinkId=512156</description>
```

Comment:
You are right. I just check that warning is gone. I will swap it.

---
In reply to: [76338357](https://github.com/Microsoft/ApplicationInsights-dotnet/pull/297#discussion_r76338357) [](ancestors = 76338357)